jpeg/jpeg_static.go

```diff
@@ -0,0 +1,21 @@
+//go:build !dynamic
+
+package jpeg
+
+/*
+Note the contents of include/ are extracted from jpeg-turbo 2.1.5.1.
+Contents of lib/ have been built from that source as well.
+Libraries are actually copies of libturbojpeg.a as built for each architecture.
+*/
+
+/*
+#cgo linux, darwin CFLAGS: -I${SRCDIR}/include
```

Comment:
```suggestion
#cgo linux,darwin CFLAGS: -I${SRCDIR}/include
```
Can't have spaces here.

Reply:
ok!  changing this